### ISSUE TITLE
Add music to game

### ISSUE THREAD

**Issue by daniyalb** [utmgdsc/dino-game#8]
Add music that plays while the game is running. Music should be copyright free.

**Comment by nitic04** [comment]
For the background music, can it be royalty free? For the music I want to use, the creator just needs to be credited, so I'm thinking of putting a credit in the readme. Is that okay?

**Comment by nitic04** [comment]
Also can someone assign this issue to me please :)

**Comment by ggggg** [comment]
> For the background music, can it be royalty free? For the music I want to use, the creator just needs to be credited, so I'm thinking of putting a credit in the readme. Is that okay?

Yes, it has to have a suitable license for us to use. Putting the credits in readme.md is perfect!
